usaspending_api/disaster/v2/views/federal_account/loans.py

```diff
@@ -51,6 +47,34 @@ def queryset(self):
             )
         )
 
+        case_when_queries = []
+        if self.last_closed_monthly_submission_dates:
+            case_when_queries.append(
+                Q(
+                    submission__reporting_fiscal_year=self.last_closed_monthly_submission_dates[
+                        "submission_fiscal_year"
+                    ],
+                    submission__reporting_fiscal_period=self.last_closed_monthly_submission_dates[
```

Comment:
Same question as in other pull request.  Should we be summing all `transaction_obligated_amount` and final for fy `gross_outlay_amount_by_award_cpe` instead of just the ones in the current period?

Reply:
Yes --
Covid stuff is for "the lifetime of the pandemic" ... so single-FY filters go away. 
- Then wrt to TOA, it is transactional and additive (like credit card transactions) ... and we should sum up any value we see at any time (as many times in a single submission, across submissions, across periods, across FYs)
- Wrt to `gross_outlay_amount_by_award_cpe`, that's a USSGL (ledger) "balance", which holds the balance _per FY_ to date (FYTD). So you want the latest reported one in the current/unclosed FY, and the one from the last quarterly/monthly periods (Period 12) in prior years. 
- It needs to be period 12 because agencies are supposed to abide by _always_ reporting a balance if it is non-zero. So if it wasn't reported in period 12, you can assume zero balance (and basically ignore it in sums).
@kbard

Reply:
@kbard @kwhickey OK, I pushed a few changes and don't have plans for more. Moving onto another feature branch while you review